### PR TITLE
Use default values for EHYSTR if not present but Hysteresis is activated by HYST in SATOPTS

### PR DESCRIPTION
**Pull request by totto82**

### CODE REVIEW

**Comment by totto82** [comment]
jenkins build this please

File: src/opm/input/eclipse/EclipseState/Runspec.cpp [hunk 1]
```diff
@@ -344,9 +344,18 @@ EclHysterConfig::EclHysterConfig(const Opm::Deck& deck)
...
-        if (!deck.hasKeyword("EHYSTR"))
-            throw std::runtime_error("Enabling hysteresis via the HYST parameter for SATOPTS requires the "
-                                     "presence of the EHYSTR keyword");
+        if (!deck.hasKeyword("EHYSTR")) {
+            std::string msg = "Hysteresis is enabled via the HYST parameter for SATOPTS, but the EHYSTR " 
+                              "keyword is not present in the deck. \n"
+                              "Default values are used for the EHYSTR keyword.";
+
+            OpmLog::warning(msg);
+            curvatureCapPrsValue = 0.1;
```
Comment (akva2) [inline src/opm/input/eclipse/EclipseState/Runspec.cpp:353]: these defaults should and are encoded as the default member var values (excluding the model indicators).

Comment (totto82) [reply src/opm/input/eclipse/EclipseState/Runspec.cpp:353]: I just thought re-setting them here made it more clear. But I can remove it. Then the commit will just change the throw to a warning.

Comment (akva2) [reply src/opm/input/eclipse/EclipseState/Runspec.cpp:353]: i think that's better, if we want to change defaults we then only have to change in one spot. however, note that you still have to set krHystMod and pcHystMod.

**Comment by totto82** [comment]
jenkins build this please

**Comment by totto82** [comment]
@OPMUSER Any comment on this? Should this be reflected in the manual?

**Comment by totto82** [comment]
Merge?

**Pull request merged**